index.html

```diff
@@ -1,67 +1,218 @@
+            don’t hesitate to contact me.</div>
+            <div>
+                <ul class="social">
+                    <li><a href="#"><img src="/image/github.png" alt="Github"></a></li>
+                    <li><a href="#"><img src="image/linkedin.png" alt="LinkedIn"></a></li>
+                    <li><a href="#"><img src="image/thumbs.png" alt="Thumbs"></a></li>
+                    <li"><a href="#"><img src="image/twitter.png" alt="Twitter"></a></li>
+                    <li><a href="#"></a></li>
+                </ul>
+            </div>
+            <div class="group-header"><img src="image/Group.png" alt="Header Group"></div>
+    </header>
 
-        <div class="social-images">
-            <ul class="social">
-                <li class="social-li">
-                    <a href="#">
-                        <img src="image/github.png" alt="Github" class="im-age">
-                    </a>
-                </li>
-                <li class="social-li">
-                    <a href="#">
-                        <img src="image/linkedin.png" alt="LinkedIn" class="im-age">
-                    </a>
-                </li>
-                <li class="social-li">
-                    <a href="#">
-                        <img src="image/thumbs.png" alt="Thumbs" class="im-age">
-                    </a>
-                </li>
-                <li class="social-li">
-                    <a href="#">
-                        <img src="image/twitter.png" alt="Twitter" class="im-age">
-                    </a>
-                </li>
-                <li class="social-li">
-                    <a href="#">
-                        <img src="image/M.png" alt="M" class="im-age">
-                    </a>
-                </li>
-            </ul>
+    <main class="main">
+        <div class="recent">My Recent Works</div>
+        <div class="grid">
+            <div class="workswidth">
+                <div class="works1">
+                    <div class="yogalady"> <img src="image/yogalady.png" alt="Yogalady"></div>
+                    <div class="multi-post1">Multi-Post Stories</div>
+                    <div class="para-works1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</div>
+                    <div class="list1">
+                        <ul class="boxes1">
+                            <li class="box box1"><img src="image/css1.png" alt="CSS"></li>
+                            <li class="box box2"><img src="image/html1.png" alt="HTML"></li>
+                            <li class="box box3"><img src="image/bootstrap1.png" alt="Bootstrap"></a></li>
+                            <li class="box box4"><img src="image/ruby1.png" alt="Ruby"></li>
+                        </ul>
+                    </div>
+                    <div><button class="button1">See Project</button></div>
```

Comment:
Great job with the first card of your project, however, I have a couple of small suggestions to make it look better and close to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-27&t=ZJcVC8XUqegdTgGs-0):

- [ ] I noticed that the text of the button is not aligned to the center, as per the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md). Would you be able to adjust the position of the button text to the center of the button? 🤓
- [ ] In addition to centering the text of your button, it is also required to add hover, active, and disabled effects to your button, following the format and style of the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-27&t=ZJcVC8XUqegdTgGs-0). 👌

![image](https://user-images.githubusercontent.com/107952724/235326491-3fad41f1-ee5b-4bb6-ac96-3ea0fdfcf77b.png)